.circleci/config.yml

```diff
@@ -101,6 +101,121 @@ jobs:
+      - run:
+          name: Install the queue container
+          command: |
+            set -eux
+            sed -e "s/CONTAINER_IP/${QUEUE_IP}/g" .circleci/cluster/queue_config.yaml > queue_config.yaml
+            docker cp queue_config.yaml ${QUEUE_NAME}:/etc/cloudify/config.yaml
+            docker cp queue_key.pem ${QUEUE_NAME}:/etc/cloudify/queue_key.pem
+            docker cp queue_cert.pem ${QUEUE_NAME}:/etc/cloudify/queue_cert.pem
+            docker exec ${QUEUE_NAME} cfy_manager install
+      - run:
+          name: Install the DB container
+          command: |
+            set -eux
+            sed -e "s/CONTAINER_IP/${DB_IP}/g" .circleci/cluster/db_config.yaml > db_config.yaml
+            docker cp db_config.yaml ${DB_NAME}:/etc/cloudify/config.yaml
+            docker cp db_cert.pem ${DB_NAME}:/etc/cloudify/cert.pem
+            docker cp db_key.pem ${DB_NAME}:/etc/cloudify/key.pem
+            docker cp ca_cert.pem ${DB_NAME}:/etc/cloudify/ca.pem
+            docker exec ${DB_NAME} cfy_manager install
+      - run:
+          name: Install the first manager container
+          command: |
+            set -eux
+            sed -e "s/CONTAINER_IP/${MANAGER1_IP}/g" \
+              -e "s/QUEUE_IP/${QUEUE_IP}/g" \
+              -e "s/DB_IP/${DB_IP}/g" \
+              .circleci/cluster/manager1_config.yaml > manager1_config.yaml
+            docker cp manager1_config.yaml ${CONTAINER_NAME}_1:/etc/cloudify/config.yaml
+
+            set +x
+            echo ${TEST_LICENSE} | sed -e 's/\\n/\n/g' > license.yaml
+            set -x
+
+            docker cp license.yaml ${MANAGER1_NAME}:/etc/cloudify/license.yaml
+            docker cp ca_cert.pem ${MANAGER1_NAME}:/etc/cloudify/ca.pem
+            docker cp db_client_1_key.pem ${MANAGER1_NAME}:/etc/cloudify/postgres_client_key.pem
+            docker cp db_client_1_cert.pem ${MANAGER1_NAME}:/etc/cloudify/postgres_client_cert.pem
+
+            docker exec ${MANAGER1_NAME} cfy_manager install
+      - run:
+          name: install manager2
+          command: |
+            set -eux
+            sed -e "s/CONTAINER_IP/${MANAGER2_IP}/g" \
+              -e "s/QUEUE_IP/${QUEUE_IP}/g" \
+              -e "s/DB_IP/${DB_IP}/g" \
+              -e "s/ACTIVE_MANAGER_IP/${MANAGER1_IP}/g" \
+              .circleci/cluster/manager2_config.yaml > manager2_config.yaml
+            docker cp manager2_config.yaml ${MANAGER2_NAME}:/etc/cloudify/config.yaml
+
+            docker cp ca_cert.pem ${MANAGER2_NAME}:/etc/cloudify/ca.pem
+            docker cp db_client_2_key.pem ${MANAGER2_NAME}:/etc/cloudify/postgres_client_key.pem
+            docker cp db_client_2_cert.pem ${MANAGER2_NAME}:/etc/cloudify/postgres_client_cert.pem
+
+            docker exec ${MANAGER2_NAME} cfy_manager install
+      - run:
+          name: status
+          command: |
+            set -eux
+            docker exec ${MANAGER1_NAME} cfy cluster status
+            docker exec ${MANAGER2_NAME} cfy cluster status
```

Comment:
Why run on both?

Reply:
i guess it's a test that both managers work